src/main/java/de/tum/in/www1/artemis/service/StaticCodeAnalysisService.java

```diff
@@ -142,6 +142,32 @@ public Set<StaticCodeAnalysisCategory> resetCategories(ProgrammingExercise exerc
         return categories;
     }
 
+    /**
+     * This method allows users to reuse an already existing SCA configuration by copying it into another exercise.
+     * The previous configuration of the targeted exercise will get removed.
+     * @param targetExercise The exercise into which the configuration gets copied in
+     * @param sourceExercise The exercise to take the existing configuration from
+     * @return the new SCA configuration of the targetExercise
+     */
+    public Set<StaticCodeAnalysisCategory> importCategoriesFromExercise(ProgrammingExercise targetExercise, ProgrammingExercise sourceExercise) {
```

Comment:
```suggestion
    public Set<StaticCodeAnalysisCategory> importCategoriesFromExercise(ProgrammingExercise sourceExercise, ProgrammingExercise targetExercise) {
```
Personal preference: Copying from source to target and therefore specifying the parameters in that order would be more intuitive to me. Can of course be left as-is, though.